### PR TITLE
Fix WebAuthn not working

### LINKED ISSUE #2154: WebAuthn Option Disappeared in Login screen in v5
### Detailed description of the problem [REQUIRED]
After upgrading from 4.13.0 to 5.0.0, I can no longer see webauthn option in login popup. In 4.13.0, I used to see a key icon in the top left corner of the login popup and was able to login with biometric mechanism, but that button is no longer there and I can't find it anywhere else.

Tried it again after upgrading from 5.0.0 to 5.0.1, and to 5.0.2 and it's still not there.

### Browser and system
Chrome and Safari on macOS Sonoma.

### PR DESCRIPTION
**Pull request by ildyria**
Fixes #2154

### CODE REVIEW

File: lang/en/lychee.php [hunk 1]
```diff
@@ -315,6 +315,7 @@
 	'U2F_AUTHENTIFICATION_SUCCESS' => 'Authentication successful!',
 	'U2F_CREDENTIALS' => 'Credentials',
 	'U2F_CREDENTIALS_DELETED' => 'Credentials deleted!',
+	'U2F_LOGIN' => 'Login with WebAuthn',
```
Comment (kamil4) [inline lang/en/lychee.php]: With my "grammar Nazi" hat on, I believe this should actually be:

```suggestion
	'U2F_LOGIN' => 'Log in with WebAuthn',
```

(verb vs noun)

File: resources/js/data/webauthn/loginWebAuthn.ts [hunk 2]
```diff
@@ -48,5 +49,29 @@ export const loginWebAuthn = (Alpine: Alpine) =>
 					})
 					.catch(() => alpine.$dispatch("notify", [{ type: "error", msg: this.error_msg }]));
 			},
+
+			handleKeyUp(event: KeyboardEvent): void {
+				const skipped = ["TEXTAREA", "INPUT", "SELECT"];
+
+				if (document.activeElement !== null && skipped.includes(document.activeElement.nodeName)) {
+					console.log("WAuthn skipped: " + document.activeElement.nodeName);
```
Comment (kamil4) [inline resources/js/data/webauthn/loginWebAuthn.ts]: A leftover?

File: resources/views/components/webauthn/login.blade.php [hunk 3]
```diff
@@ -2,8 +2,8 @@
     bg-black/80 z-5 fixed flex items-center justify-center w-full h-full top-0 left-0 box-border opacity-100"
     data-closable="true"
     x-data="loginWebAuthn('{{ __("lychee.U2F_AUTHENTIFICATION_SUCCESS") }}', '{{ __("lychee.ERROR_TEXT") }}')"
-    @keydown.window="if (event.key === 'k') { webAuthnOpen = true }"
-    @keydown.escape.window="webAuthnOpen = false"
+    @keyup.window="handleKeyUp(event)"
+    x-on:webauthn-open.document="webAuthnOpen = true; console.log('open')"
```
Comment (kamil4) [inline resources/views/components/webauthn/login.blade.php]: Another leftover debug log?

File: resources/js/data/webauthn/loginWebAuthn.ts (hunk 2 above)
Comment (ildyria) [reply resources/js/data/webauthn/loginWebAuthn.ts]: not really. But I can remove it :)

File: resources/views/components/webauthn/login.blade.php (hunk 3 above)
Comment (ildyria) [reply resources/views/components/webauthn/login.blade.php]: that one yes. :)

File: resources/js/data/webauthn/loginWebAuthn.ts (hunk 2 above)
Comment (kamil4) [reply resources/js/data/webauthn/loginWebAuthn.ts]: I think it depends what triggers it (and right now I don't even pretend to understand that).

If it's more of an assertion that Just Shouldn't Happen (TM) then I think it's fine to keep it (perhaps after replacing "WAuthn" with "WebAuthn", to be consistent). If on the other hand this message triggers during regular use, then I would remove it -- I don't think we should be printing to the debug log unless something rather unexpected is happening.

Comment (ildyria) [reply resources/js/data/webauthn/loginWebAuthn.ts]: That small if statement prevents to trigger keybinding behaviors when you are text areas etc...

File: lang/en/lychee.php (hunk 1 above)
Comment (ildyria) [reply lang/en/lychee.php]: Fixed in [cba3849](https://github.com/LycheeOrg/Lychee/pull/2155/commits/cba3849efb6514c55b72d865a60d1961525506de)

File: lang/de/lychee.php [hunk 4]
```diff
@@ -315,6 +315,7 @@
 	'U2F_AUTHENTIFICATION_SUCCESS' => 'Authentifizierung erfolgreich!',
 	'U2F_CREDENTIALS' => 'Anmeldedaten',
 	'U2F_CREDENTIALS_DELETED' => 'Anmeldedaten gelöscht!',
+	'U2F_LOGIN' => 'Log in with WebAuthn',
```
Comment (qwerty287) [inline lang/de/lychee.php]:
```suggestion
	'U2F_LOGIN' => 'Mit WebAuthn anmelden',
```

**Review by qwerty287 — APPROVED** [review]

**Pull request merged**